doc/zero-to-binderhub/setup-prerequisites.rst

```diff
@@ -3,152 +3,62 @@
 Set up the prerequisites
 ========================
 
-BinderHub is built to run on top of Kubernetes, a distributed cluster manager.
-It uses a JupyterHub to launch and manage user servers, as well as a
-docker registry to cache images.
+BinderHub is built to run in a `Kubernetes cluster <http://kubernetes.io/>`_. It
+relies on JupyterHub to launch and manage user servers, as well as a docker
+registry to cache docker images it builds.
 
-To create your own BinderHub, you'll first need to set up a Kubernetes Cluster
-and then configure its various components correctly. The following instructions
-will assist you in doing so.
-
-.. note::
-
-   BinderHub uses a JupyterHub running on Kubernetes for much of its functionality.
-   For information on setting up and customizing your JupyterHub, we recommend reading
-   the `Zero to JupyterHub Guide <https://zero-to-jupyterhub.readthedocs.io/en/latest/index.html>`_.
+To deploy your own BinderHub, you'll first need to set up a Kubernetes cluster.
+The following instructions will assist you in doing so.
 
 Setting up a Kubernetes cluster
 -------------------------------
 
-.. note::
-
-   BinderHub is built to be cloud agnostic, and can run on various cloud
-   providers (as well as bare metal). If you would like to help with adding
-   instructions for other cloud providers or improving the current instructions,
-   `please contact us <https://github.com/jupyterhub/zero-to-jupyterhub-k8s/issues>`_!
-
-First, deploy a Kubernetes cluster by following the
-`instructions in the Zero to JupyterHub guide <https://zero-to-jupyterhub.readthedocs.io/en/latest/create-k8s-cluster.html>`_.
+First, deploy a Kubernetes cluster by following the `instructions in the Zero to
+JupyterHub guide
+<https://zero-to-jupyterhub.readthedocs.io/en/latest/kubernetes/setup-kubernetes.html>`_.
 When you're done, move on to the next section.
 
-
 Installing Helm
```

Comment:
For the installing Helm section, why don't we also just link to the z2jh docs in order to minimise replication and number of places to maintain?

Reply:
Installing Helm 3 is trivial, only a binary needs to be installed at this point, so ensuring the documentation is in sync with z2jh is an about as complicated maintenance I think. But we if anyone wants to do this later I'd be fine with it.